packages/flutter/lib/src/material/material_app.dart

```diff
@@ -141,8 +141,9 @@ class _MaterialAppState extends State<MaterialApp> implements BindingObserver {
       data: new MediaQueryData(size: _size),
       child: new LocaleQuery(
         data: _localeData,
-        child: new Theme(
+        child: new AnimatedTheme(
```

Comment:
How is the performance of animating the theme?  I would imagine it to be pretty bad.  I guess we can worry about speeding it up separately.